GeoQuizChallenge/build.gradle

```diff
@@ -1,18 +1,37 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
     repositories {
         google()
         jcenter()
+        gradlePluginPortal()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.5.2'
-
+        classpath 'io.gitlab.arturbosch.detekt:detekt-gradle-plugin:1.0.1'
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
     }
 }
 
+apply plugin: 'io.gitlab.arturbosch.detekt'
+
+//plugins {
```

Comment:
maybe add a TODO?

Reply:
i think this is not needed anymore so maybe we can delete it?

Reply:
It's needed!